_build_cfg.yml

```diff
@@ -391,6 +391,9 @@ Topics:
   - Name: Scoped Tokens
     File: scoped_tokens
     Distros: openshift-origin
+  - Name: Monitoring Images
+    File: manage_images
```

Comment:
@soltysh Sorry! One more thing. Can you please update the file name to match the title? monitoring_images instead of manage_images.